app/services/websocket_server.py

```diff
@@ -202,24 +253,47 @@ def format_order(record):
     token = data["token"] if "token" in data and Web3.isAddress(data["token"]) else None
     user = data["user"] if "user" in data and Web3.isAddress(data["user"]) else None
 
+    # response vars
     trades = []
     my_trades = []
     my_funds = []
+    tickers = {}
+    
+    # if token is passed in
     if token:
+        
+        # get ticker for passed in token
+        #tickers = await get_tickers(token)
+        tickers = await get_tickers()
```

Comment:
We could probably extract that to above if we no longer differentiate by token.

Reply:
Fixed.